### ISSUE TITLE
Taking a close look at CRS3 performance against WAVSEP

### ISSUE THREAD

**Issue by fzipi** [crstest01/owasp-modsecurity-crs#690]
_Issue originally created by user dune73 on date 2017-02-09 13:16:53.
            Link to original issue: https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/685._

The Web Application Vulnerability Scanner Evaluation Project at 
https://github.com/sectooladdict/wavsep/
seems to be the tool of choice to determine the quality of a scanner. This due to the fact that the density of vulnerabilities is very high.

I am currently looking into this together with a friend. Creating this as an issue and adding labels, even if they do not really fit.

**Comment by fzipi** [comment]
_User csanders-git commented on date 2017-02-23 23:27:40:_

**at user dune73** how is this going?

**Comment by fzipi** [comment]
_User dune73 commented on date 2017-02-24 05:20:42:_

The test server is configured, testing is about to start.

**Comment by fzipi** [comment]
_User dune73 commented on date 2017-02-24 21:02:02:_

Setup description: https://www.netnea.com/cms/2017/02/24/optimized-lab-setup-to-attack-modsecurity-with-security-sanners/

**Comment by fzipi** [comment]
_User csanders-git commented on date 2017-03-06 15:24:41:_

I like how you changed the paranoia level as part of a request... perhaps we should add this to the regression tests

**Comment by fzipi** [comment]
_User dune73 commented on date 2017-03-06 16:26:46:_

It's quite neat, thanks. There is also a session-tracking field in the logformat that I advocate. Setting this via another X-header lets us group it all very easily. Btw. burp scanner had a detection rate of 97% in PL4. What is open can mostly not be considered an attack in itself. (Reconnaissance or specific weaknesses in the backend triggered by certain innocent requests)

**Comment by fzipi** [comment]
_User csanders-git commented on date 2017-03-06 19:32:59:_

**at user dune73** wow, that's great! i'm glad this project is being undertaken formally

**Comment by fzipi** [comment]
_User sathieu commented on date 2018-02-14 12:34:29:_

Hello, Any news on this?

**Comment by fzipi** [comment]
_User dune73 commented on date 2018-02-14 13:10:13:_

No, not really. My research partner dropped out and he never published the results of the part that we finished. However, I have the data for burp (at hand!) and I stand by the data behind one slide that I use in my presentations: https://de.slideshare.net/ChristianFolini/introducing-the-owasp-modsecurity-core-rule-set-75913263, slide number 9.

I expect the results of the other scanners to be similar, and it's a pity we did not finish this nor formally publish.
